From e817f08b7a4026593a85953037e19027a094b0ba Mon Sep 17 00:00:00 2001
From: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Date: Fri, 22 Nov 2024 17:45:28 +0100
Subject: [PATCH 1388/1695] dts: stm32: add OEM rproc public key hash for
 stm32mp21

Add new OTP entry for remoteproc authentication key hash.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 64696d72c91652164704fbc105baa447d7e89844 dts: stm32: add OEM rproc public key hash for stm32mp21]
Change-Id: If7f2617794cf4cab67c97da95539d9e75c396d76
Signed-off-by: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/420444
Tested-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 55dc3aecc..08049ad07 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -2454,6 +2454,12 @@
 				reg = <0x260 0x20>;
 			};
 
+			oem_rproc_pkh: oem_rproc_pkh@2d0 {
+				/* 0x2D0 -> otp180 */
+				reg = <0x2D0 0x20>;
+				st,non-secure-otp-provisioning;
+			};
+
 			oem_fip_enc_key: otp260@410 {
 				reg = <0x410 0x20>;
 				st,non-secure-otp-provisioning;
-- 
2.47.2

